modules/game/src/main/Game.scala

```diff
@@ -116,7 +116,7 @@ case class Game(
    * It starts at 1, and is incremented after Black's move.
    * NOTE: Duplicates chess.Game.fullMoveNumber (avoids loading toChess)
    */
-  def fullMoveNumber: Int = 1 + turns / 2
+  def fullMoveNumber: Int = 1 + (turns - 1) / 2
```

Comment:
This value is only used in widgets.template.scala 93.47 (it's the only usage), the original field that this duplicates remains unchanged.